### PR TITLE
LPS-68390 Alphabetical order

### PR DESCRIPTION
**Pull request by brunobasto**

### CODE REVIEW

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js [hunk 1]
```diff
@@ -425,9 +441,18 @@ AUI.add(
...
+											var responseData = this.get('responseData');
+
+											instance._defineIds(responseData);
 
 											instance.savedState = state;
+
+											instance.fire(
+												'autosave',
+												{
+													modifiedDate: responseData.modifiedDate
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js]: @brandizzi, can you take a look at this? I'm worried about the displayed date format and timezone issues.

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Pull request submitted to https://github.com/brunobasto/liferay-portal/pull/782. [See changes here.](https://github.com/brandizzi/liferay-portal/compare/be5c74d~4...be5c74d) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**